Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+
+            var scriptPath = Path.GetFullPath("Packages/com.unity.meshsync/Editor/Scripts/DCCIntegration/Launchers/blender_interop.py");
+
+            startInfo.Arguments = $"\"{absoluteAssetPath}\" -P \"{scriptPath}\"";
+
+            if (runMode != RunMode.GUI)
+                startInfo.Arguments = "-b " + startInfo.Arguments;
+
+            if (runMode == RunMode.Background)
+            {
+                startInfo.UseShellExecute = false;
+                startInfo.CreateNoWindow = true;
+                startInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
+            }
+
+            m_blenderProcess = new System.Diagnostics.Process();
+
+            if (redirectBlenderToUnityConsole)
+            {
+                startInfo.RedirectStandardOutput = true;
+                startInfo.RedirectStandardError = true;
+
+                m_blenderProcess.ErrorDataReceived -= M_ProcessOnErrorDataReceived;
+                m_blenderProcess.OutputDataReceived -= M_ProcessOnOutputDataReceived;
+
+                m_blenderProcess.ErrorDataReceived += M_ProcessOnErrorDataReceived;
+                m_blenderProcess.OutputDataReceived += M_ProcessOnOutputDataReceived;
+
+                startInfo.UseShellExecute = false;
+            }
+
+            m_blenderProcess.StartInfo = startInfo;
+
+            m_blenderProcess.EnableRaisingEvents = true;
+            m_blenderProcess.Exited += M_blenderProcess_Exited;
+
+            m_blenderProcess.Start();
+
+            if (redirectBlenderToUnityConsole)
+            {
+                m_blenderProcess.BeginOutputReadLine();
+                m_blenderProcess.BeginErrorReadLine();
+            }
+        }
+
+        public void DrawDCCToolVersion(BaseMeshSync player)
+        {
+            GUILayout.BeginVertical("Box");
+
+            if (player is MeshSyncServer server)
+            {
+                var blenderPath = GetBlenderPath();
+
+                GUILayout.BeginHorizontal();
+                var style = new GUIStyle(GUI.skin.label) { wordWrap = true };
+                EditorGUILayout.LabelField("Blender path:", blenderPath, style);
+                if (GUILayout.Button("...", GUILayout.Width(30)))
+                {
+                    var newBlenderPath = EditorUtility.OpenFilePanel("Blender path", Path.GetDirectoryName(blenderPath), "exe");
+
+                    if (!string.IsNullOrEmpty(newBlenderPath) && newBlenderPath != blenderPath)
+                    {
+                        SetBlenderPath(newBlenderPath);
+                        OpenDCCTool(server.m_DCCAsset);
+                    }
+                    GUILayout.EndVertical();
+                    return;
+                }
+
+                GUILayout.EndHorizontal();
+
+                GUILayout.BeginHorizontal();
+
+                var newRunMode = (RunMode)EditorGUILayout.EnumPopup("Run mode:", runMode);
```

Comment:
Why would a user open Blender in Background or Console mode ?

Reply:
Performance mostly, blender runs faster without the UI.